platforms/documentation/docs/src/docs/userguide/jvm/scala_plugin.adoc

```diff
@@ -216,7 +216,7 @@ _Default value:_ `[__projectDir__/src/__name__/scala]`.
 `allScala` — link:{javadocPath}/org/gradle/api/file/FileTree.html[FileTree] (read-only)::
 All Scala source files of this source set. Contains only the `.scala` files found in the Scala source directories. _Default value:_ non-null.
 
-These extensions are backed by an object of type link:{groovyDslPath}/org.gradle.api.plugins.scala.ScalaSourceSet.html[ScalaSourceSet].
+These extensions are backed by an object of type link:{javadocPath}/org/gradle/api/tasks/ScalaSourceSet.html[ScalaSourceSet].
```

Comment:
💅 `ScalaSourceSet` is actually deprecated, maybe the documentation here should be updated.

> Using conventions to contribute source sets is deprecated. You can configure the groovy sources via the `ScalaSourceDirectorySet` extension (e.g.
> `sourceSet.getExtensions().getByType(ScalaSourceDirectorySet.class).setSrcDirs(...)`). This interface is scheduled for removal in Gradle 9.0.

Reply:
These pages are due for an update in a future PR.